wfm

```diff
@@ -142,6 +148,16 @@ clean () {
   done
 }
 
+buildTemplates () {
```

Comment:
thought about it, but its really just a failsafe as we should always have built dist in master.